src/it/example-audit-execution/pom.xml

```diff
@@ -54,6 +55,7 @@
             <id>execute-audits</id>
             <goals>
               <goal>audit</goal>
+              <goal>excel</goal>
```

Comment:
Let's add to the README for the project that the `excel` goal uses the `poi` library under-the-hood which has some characteristics users of the plugin may want to be aware of:

1. The build may fail from a `poi` exception (`ArrayIndexOutOfBounds` and similar); when this happens, running the build again will often produce the expected result
2. The build may become inefficient in the excel goal with large-scale datasets (e.g. those with 100k or more `AuditResult` entries)

In future, we will include a `csv` goal instead that can be used for Excel and either retire the excel goal or keep both